workshop/posting-errors-guidelines.md

```diff
@@ -140,56 +140,55 @@ First, navigate to your particular workshop's training channel.
 To keep new questions and responses from getting lost in the messaging board, we strongly encourage you to make a "headline" that is one post and put the longer description you crafted in step 2 in the thread of your headline.
 
 *For our example:*
-<img src = "screenshots/error-headline.png" width = "600">
+<img src = "screenshots/error-headline.png" alt = "Error headline" width = "600">
 
 To navigate to Slack threads, you can can click speech bubble in the corner of your post:
 
-<img src = "screenshots/thread-button.png" width = "200">
+<img src = "screenshots/thread-button.png" alt = "Thread button" width = "200">
 
 You should try to keep all further messages and correspondence about that question in the same Slack thread.
 Follow up problems in the same section of code should also be kept to this same thread you started.
 
 #### Step 3b) In the thread of your headline, post the description you crafted in [Step 2](#step-2-draft-out-your-question-youll-post).
 
-Navigate to the Slack thread of your "headline".
-You can can click speech bubble in the corner of your post: <img src = "screenshots/thread-button.png" width = "150">
+Navigate to the Slack thread of your "headline" and click the speech bubble in the corner of your post.
```

Comment:
This still feels redundant? I don't think I mind it as much as I would have with the double image though, so I'm glad you caught that!

Reply:
Changing to `Navigate to the Slack thread of your "headline" post.`